be/src/util/bit_util.h

```diff
@@ -300,6 +300,11 @@ class BitUtil {
         return (value + (factor - 1)) & ~(factor - 1);
     }
 
+    // speed up function compute for SIMD
+    static inline size_t RoundUpToPowerOf2Int32(size_t value, size_t factor) {
+        return (value + (factor - 1)) & ~(factor - 1);
```

Comment:
What's the keypoint for auto SIMD here?
Just remove ```DCHECK``` is ok?

Reply:
1. keypoint is `inline`.
GCC can do auto SIMD in this `for loop`:
`        auto mem_size = 0;
        for (int i = 0; i < max_fetch; ++i) {
            mem_len[i] = (value + (8 - 1)) & ~(8 - 1);
            mem_size += mem_len[i];
        }
`

2. I will Add DCHECK here